src/goto-instrument/contracts/doc/user/contracts-assigns.md

```diff
@@ -500,6 +500,79 @@ int foo()
 }
 ```
 
+## Loop Assigns Inference
+
+When loop invariants clauses are specified but loop assigns are not, CBMC will infer
+loop assigns clauses and use them to apply loop contracts. The idea of the inference
+is to include all the left hand side of assignments in the loop.
```

Comment:
What if a reference is taken as a r-value. i.e. a function call that contains f(&x, b) - is that an assign to x?